packages/create-package/README.md

```diff
@@ -10,13 +10,19 @@ Provides an `npm init` script for creating new "UI5 Web Components" packages.
 
 with `npm`
 
-- `npm init @ui5/webcomponents-package` (the user will be asked for **name**, **port**, **tag**);
-- `npm init @ui5/webcomponents-package <NEW-PACKAGE-NAME>` (the user will be asked for **port** and **tag** only).
+- `npm init @ui5/webcomponents-package` (the user will be asked for **name**, **port**, **tag**, **typescript support**);
+- `npm init @ui5/webcomponents-package <NEW-PACKAGE-NAME>` (the user will be asked for **port**, **tag** and **typescript support** only).
+- `npm init @ui5/webcomponents-package <NEW-PACKAGE-NAME> <NEW-PACKAGE-PORT>` (the user will be asked for **tag** and **typescript support** only).
+- `npm init @ui5/webcomponents-package <NEW-PACKAGE-NAME> <NEW-PACKAGE-PORT> <NEW-PACKAGE-TAG>` (the user will be asked for **typescript support** only).
+- `npm init @ui5/webcomponents-package <NEW-PACKAGE-NAME> <NEW-PACKAGE-PORT> <NEW-PACKAGE-TAG> <NEW-PACKAGE-TYPESCRIPT-SUPPORT>` (the user will not be asked for anything).

```

Comment:
use `yargs` as in dev-server.js, more developer friendly than relying on parameter order